packages/image_picker/lib/image_picker.dart

```diff
@@ -15,4 +15,10 @@ class ImagePicker {
     final String path = await _channel.invokeMethod('pickImage');
     return new File(path);
   }
+
+  // Return bool for store
```

Comment:
Comment should be /// (dartdoc) and could use improvement. We should make a note if it only works on iOS, or ideally implement Android at the same time.

I think this should be `Future<Null>` and complete with an error if the store fails.